Modules/_uuidmodule.c

```diff
@@ -13,10 +19,23 @@ py_uuid_generate_time_safe(void)
     res = uuid_generate_time_safe(out);
     return Py_BuildValue("y#i", (const char *) out, sizeof(out), res);
 }
+#else
+static PyObject *
+py_uuid_generate_time(void)
+{
+    uuid_t out;
+    uuid_generate_time(out);
+    return PyBytes_FromStringAndSize((const char *) out, sizeof(out));
```

Comment:
I'd prefer if you'd return the `(out, None)` tuple here.

Reply:
I prefer to have very thin wrapper for C functions, and handle differences at the Python level.

Reply:
IMO it only makes both the Python code and the C code more complicated.

Reply:
One of the annoyances with modifying `uuid.py` is that there are so different code paths to take care of.